pkg/stub/status.go

```diff
@@ -83,12 +81,10 @@ func (h *Handler) syncOperatorStatus() {
 
 	if isNotFound {
 		if err := sdk.Create(co); err != nil {
-			logrus.Errorf("syncOperatorStatus: failed to create ClusterOperator %s: %v",
-				co.Name, err)
-		} else {
-			logrus.Infof("syncOperatorStatus: created ClusterOperator %s (UID %v)",
-				co.Name, co.UID)
+			logrus.Errorf("syncOperatorStatus: failed to create ClusterOperator %s: %v", co.Name, err)
 		}
+		logrus.Infof("syncOperatorStatus: created ClusterOperator %s (UID %v)", co.Name, co.UID)
```

Comment:
Need a `return` after the `logrus.Errorf`.

Reply:
https://github.com/openshift/cluster-dns-operator/pull/83